modules/heat_conduction/test/tests/NAFEMS/transient/T3/nafems_t3_edge_template.i

```diff
@@ -0,0 +1,87 @@
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 5
+  xmin = 0.0
+  xmax = 0.1
+  elem_type = EDGE2
+[]
+
+[Variables]
+  [./temp]
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = 0.0
+  [../]
+[]
+
+[BCs]
+  [./FixedTempLeft]
+    type = DirichletBC
+    variable = temp
+    boundary = left
+    value = 0.0
+  [../]
+  [./FunctionTempRight]
+    type = FunctionDirichletBC
+    variable = temp
+    boundary = right
+    function = '100.0 * sin(pi*t/40)'
+  [../]
+[]
+
+[Kernels]
+  [./heat]
+    type = HeatConduction
+    variable = temp
+  [../]
+  [./HeatTdot]
+    type = HeatConductionTimeDerivative
+    variable = temp
+  [../]
+[]
+
+[Materials]
+  [./density]
+    type = GenericConstantMaterial
+    prop_names = 'thermal_conductivity specific_heat density'
+    prop_values = '35.0 440.5 7200.0'
+  [../]
+[]
+
+[Preconditioning]
```

Comment:
This block doesn't have any effect on this simulation because there's a single variable.  I'd suggest removing it to trim down the input.

Reply:
Done.